app/src/docs/components/DocsPages/ModuleReference/Text/index.jsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable global-require, import/no-dynamic-require, react/no-array-index-key */
+// @flow
+
+import React from 'react'
+import { Helmet } from 'react-helmet'
+import CanvasModuleHelp from '$docs/components/CanvasModuleHelp'
+import DocsLayout from '$docs/components/DocsLayout'
+import docsStyles from '$docs/components/DocsLayout/docsLayout.pcss'
+import { canvasModulesCategorised } from '../data'
+
+export default function TextModules() {
+    return (
+        <DocsLayout>
+            <Helmet title="Text Modules | Streamr Docs" />
+            {(// $FlowFixMe
+                Object.entries(canvasModulesCategorised).map(([category, helps]: [string, any[]]) => (
```

Comment:
Does this need to do any looping at all if it's just looping to find a single entry? i.e.

```JS
export default function TextModules() {
    const category = 'Text'
    const helps = canvasModulesCategorised[category]
    return (
        <DocsLayout>
            <Helmet title="Text Modules | Streamr Docs" />
            <section key={category} className={docsStyles.canvasModule}>
                <h1 className={docsStyles.mdH1}>{category}</h1>
                    {helps.map((m) => (
                        <CanvasModuleHelp key={m.id} module={m} help={m.help} />
                     ))}
            </section>
        </DocsLayout>
    )
}
```

And then if `category` was a prop, all of these individual category listing components could use a single component:

```JS
export default function HelpModules({ category, pageTitle }) {
    const helps = canvasModulesCategorised[category]
    return (
        <DocsLayout>
            <DocsHelmet title={pageTitle} />
            <section key={category} className={docsStyles.canvasModule}>
                <h1 className={docsStyles.mdH1}>{category}</h1>
                    {helps.map((m) => (
                        <CanvasModuleHelp key={m.id} module={m} help={m.help} />
                     ))}
            </section>
        </DocsLayout>
    )
}
```

Reply:
Good point! I didn't look too much into this at the time, this approach seems like a time saving refactor.